packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -111,6 +146,19 @@ export const FileSettingsFields = ({ isImage, folderId }: SettingsFormProps): Re
                     name="title"
                     component={FinalFormInput}
                     fullWidth
+                    endAdornment={
+                        ContentGenerationEnabled && (
+                            <IconButton
+                                sx={{ color: theme.palette.primary.main }}
+                                onClick={async () => {
+                                    const { data } = await generateImageTitle({ variables: { imageUrl: file.fileUrl } });
```

Comment:
Also, I'd prefer renaming `imageUrl` to `fileUrl` because both terms have a meaning in COMET. fileUrl = directly from the server, imageUrl = routed through imgproxy